src/cockpit/beiboot.py

```diff
@@ -319,6 +319,10 @@ async def connect_from_bastion_host(self) -> None:
         if not ssh_askpass.exists():
             logger.error("Could not find cockpit-askpass helper at %r", askpass)
 
+        env_known_hosts = os.getenv('COCKPIT_SSH_KNOWN_HOSTS_FILE')
+        if env_known_hosts is not None:
+            args += ['-o', f'GlobalKnownHostsFile={env_known_hosts}']
```

Comment:
does this need escaping of some sort?

Reply:
Certainly not escaping -- there is no shell involved, this is an argv string.

At most this could need *un*escaping. But [cockpit-ssh](https://github.com/cockpit-project/cockpit/blob/f540cfaef30ae50f4f9fc2e2e1aa7b35623804a0/src/ssh/cockpitsshoptions.c#L100) takes the value verbatim, and I don't want to change the behaviour (and thus break existing setups). It's also rather unusual to have funny names as container volumes -- so far nobody complained yet that they need an escaping mechanism in these env vars.

Reply:
> Certainly not escaping -- there is no shell involved, 

Yeah, but ssh needs to parse the "foo=bar" string somehow, and maybe it does some unescaping. Anyway, no need here.